syft/workers/base.py

```diff
@@ -118,6 +119,47 @@ def __init__(
                         if self.id not in worker._known_workers:
                             worker.add_worker(self)
 
+    def run_script_module(self, serialized_buffer: bin):
```

Comment:
I propose to first focus on sending the object to the worker and loading it from the buffer. I don't see the need of a run_script_module() function as this would be covered by the TrainConfig developments. (Ok for testing, though :-) )

Reply:
This was for testing only :), I was checking if we were able to train a model remotely using the serialized model.